src/adapter/src/catalog/open.rs

```diff
@@ -253,19 +253,44 @@ impl Catalog {
             let is_read_only = storage.is_read_only();
             let mut txn = storage.transaction().await?;
 
-            migrate::durable_migrate(&mut txn, config.boot_ts)?;
+            // Migrate/update durable data before we start loading the in-memory catalog.
+            {
+                migrate::durable_migrate(&mut txn, config.boot_ts)?;
+                // Overwrite and persist selected parameter values in `remote_system_parameters` that
+                // was pulled from a remote frontend (if present).
```

Comment:
nit: can you toss in something like `... remote frontend (e.g. LaunchDarkly)` to this comment?

Reply:
Done